.npmrc

```diff
@@ -1 +1,2 @@
 @cahllagerfeld:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=${GH_TOKEN}
```

Comment:
In Gitpod envrionment variables can be set per project.
The idea is, that the contributor needs to have one `GH_TOKEN` in his env.

I need to test this locally if that works how I want to

Reply:
Yeah, this makes sense. Nice one!